### PR TITLE
ExpectationInterface extends Throwable

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Closes https://github.com/mockery/mockery/issues/991.

### CODE REVIEW

File: library/Mockery/ExpectationInterface.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 namespace Mockery;
 
-interface ExpectationInterface
+interface ExpectationInterface extends Throwable
```
Comment (localheinz) [inline library/Mockery/ExpectationInterface.php:24]:
```suggestion
interface ExpectationInterface extends \Throwable
```

**Comment by GrahamCampbell** [comment]
Totally wrong... 😆

**Comment by davedevelopment** [comment]
I did wonder :joy:

**Comment by GrahamCampbell** [comment]
Misread it as Exception, but it is Expectation. :trollface:

**Pull request closed without merging**